custom/icds/utils/sms.py

```diff
@@ -0,0 +1,35 @@
+import pytz
+from datetime import timedelta
+
+from corehq.apps.sms.api import get_utcnow
+from corehq.util.timezones.conversions import ServerTime
+
+
+def get_next_available_time(domain_object, domain_now):
+    if domain_object.restricted_sms_times:
+        restricted_sms_time = domain_object.restricted_sms_times[0]
```

Comment:
what if there is more than 1 window in the list?

Reply:
Agreed. Since this is happening just for ICDS right now, they just have one window.

I was thinking of a more global solution where one can consider multiple windows and use the one that is next nearest one instead but didn't want to get into that as of now.